src/test/kotlin/io/getunleash/UnleashConfigTest.kt

```diff
@@ -54,14 +54,14 @@ class UnleashConfigTest {
         assertThatThrownBy {
             UnleashConfig.newBuilder().proxyUrl("http://localhost:4242/proxy").build()
         }.isInstanceOf(IllegalStateException::class.java)
-            .hasMessage("You have to set client secret in your UnleashConfig")
+            .hasMessage("You have to set client key in your UnleashConfig")
     }
 
     @Test
     fun `Can set client timeouts in seconds with builder methods`() {
         val config = UnleashConfig(
             proxyUrl = "https://localhost:4242/proxy",
-            clientSecret = "some-secret",
+            clientKey = "some-secret",
```

Comment:
Very minor, but we might want to update the value here (and in the other tests) to indicate that it's not a secret anymore.

```suggestion
            clientKey = "some-key",
```